R/pairwise.R

```diff
@@ -8,7 +8,7 @@
 #' @param diag If similarity between itself should be returned or not.
 #' @param method Type of calculation. https://cran.r-project.org/web/packages/proxy/vignettes/overview.pdf
```

Comment:
Looks like the comments here are out of sync with current arguments of do_cosine_sim.kv.
I don't see group, dimension, and method arguments.